doc/programming/language/types.rst

```diff
@@ -290,7 +290,7 @@ Spicy provides POSIX-style regular expressions.
 
 .. rubric:: Constants
 
-- ``/Foo*ba?r/``, ``/X(..)(..)(..)Y/``
+- ``/Foo*bar?/``, ``/X(..)(..)(..)Y/``
```

Comment:
(Doesn't really matter, but not a typo.)

Reply:
This is working around `typos` considering `ba` to be a misspelling,

```.console
error: `ba` should be `by`, `be`
  --> doc/programming/language/types.rst:293:9
    |
293 | - ``/Foo*ba?r/``, ``/X(..)(..)(..)Y/``
    |          ^^
    |
```

The other place you flagged below has a similar reason. I also fixed a few places where we used `nope` and `noe` to use more common placeholder strings (`typos` seems to understand e.g., `foo` and `bar`).

I decided to work around such diagnostics instead of whitelisting whole files. At least from this initial pass that seemed acceptable; I'd suggest we try going with this and possibly using different strings where possible and only look into whitelisting if things become too annoying.